plugin.xml

```diff
@@ -59,17 +59,20 @@
 		<framework src="src/android/build.gradle" custom="true" type="gradleReference" />
 
 		<!-- Default versions for Gradle dependencies -->
-		<preference name="ANDROID_PLAY_SERVICES_AUTH_VERSION" default="18.1.0" />
-		<preference name="ANDROID_FIREBASE_ANALYTICS_VERSION" default="17.5.0" />
-		<preference name="ANDROID_FIREBASE_MESSAGING_VERSION" default="20.3.0" />
-		<preference name="ANDROID_FIREBASE_CONFIG_VERSION" default="19.2.0" />
-		<preference name="ANDROID_FIREBASE_PERF_VERSION" default="19.0.9" />
-		<preference name="ANDROID_FIREBASE_AUTH_VERSION" default="19.4.0" />
-		<preference name="ANDROID_FIREBASE_FIRESTORE_VERSION" default="21.7.0" />
-		<preference name="ANDROID_FIREBASE_CRASHLYTICS_VERSION" default="17.2.2" />
-		<preference name="ANDROID_FIREBASE_CRASHLYTICS_NDK_VERSION" default="17.2.2" />
-		<preference name="ANDROID_GSON_VERSION" default="2.8.6" />
-		<preference name="ANDROID_FIREBASE_PERF_GRADLE_PLUGIN_VERSION" default="1.3.1" />
+		<preference name="ANDROID_PLAY_SERVICES_AUTH_VERSION" default="19.0.0" />
+		<preference name="ANDROID_FIREBASE_ANALYTICS_VERSION" default="19.0.0" />
+		<preference name="ANDROID_FIREBASE_MESSAGING_VERSION" default="22.0.0" />
+		<preference name="ANDROID_FIREBASE_CONFIG_VERSION" default="21.0.0" />
+		<preference name="ANDROID_FIREBASE_PERF_VERSION" default="20.0.2" />
+		<preference name="ANDROID_FIREBASE_AUTH_VERSION" default="21.0.1" />
+		<preference name="ANDROID_FIREBASE_FIRESTORE_VERSION" default="23.0.2" />
+		<preference name="ANDROID_FIREBASE_FUNCTIONS_VERSION" default="20.0.0" />
+			<preference name="ANDROID_FIREBASE_IID_VERSION" default="21.1.0" />
+		<preference name="ANDROID_FIREBASE_INSTALLATIONS_VERSION" default="17.0.0" />
+		<preference name="ANDROID_FIREBASE_CRASHLYTICS_VERSION" default="18.1.0" />
```

Comment:
Making a note to check if we should remove our hardcoded dependencies versioning in our package.json